### ISSUE TITLE
Error in CAP generation: Application data doesn't include CDOL1 field

### ISSUE THREAD

**Issue by Meatballs1** [russss/python-emv#12]
```
Error in CAP generation: Application data doesn't include CDOL1 field: {
(57) Track 2 Equivalent Data: [], 
([5F 20]) Cardholder Name: "", 
([9F 1F]) Track 1 Discretionary Data: [], 
([5F 25]) Application Effective Date: 19/04/01, 
([5F 24]) Application Expiration Date: 22/03/31, 
(5A) Application Primary Account Number (PAN): , 
([5F 34]) Application Primary Account Number (PAN) Sequence Number: 0, 
([9F 07]) Application Usage Control: <AUC: Valid for international cash transactions>, 
(8E) Cardholder Verification Method (CVM) List: <CVM List x: 0, y: 0, rules: If terminal supports the CVM, Plaintext PIN verification performed by ICC. Else, fail verification.; If terminal supports the CVM, Enciphered PIN verified online; If terminal supports the CVM, Enciphered PIN verified online; Always, Plaintext PIN verification performed by ICC>, 
([9F 0D]) Issuer Action Code - Default: [F0 60 EC 28 00], 
([9F 0E]) Issuer Action Code - Denial: [00 10 00 00 00], 
([9F 0F]) Issuer Action Code - Online: [F0 68 FC F8 00], 
([5F 28]) Issuer Country Code: GB}
```

This seems to only have the appdata from the first time but the CDOL data is listed with `emvtool info` within the second file:

```
┌File: 2,2┬─────────────────────────────────────────────────┬─────────────────────────────────────────────────────────────────────────────────┐
│ Tag     │ Name                                            │ Value                                                                           │
├─────────┼─────────────────────────────────────────────────┼─────────────────────────────────────────────────────────────────────────────────┤
│ [9F 08] │ Application Version Number                      │ [00 8D]                                                                         │
│ [9F 42] │ Application Currency Code                       │ GBP                                                                             │
│ [5F 30] │ Service Code                                    │ [02 23]                                                                         │
│ [9F 44] │ Application Currency Exponent                   │ 2                                                                               │
│ [8C]    │ Card Risk Management Data Object List 1 (CDOL1) │ DOL([(([9F 02]) Amount, Authorised, 6), (([9F 03]) Amount, Other (Numeric), 6), │
│         │                                                 │ (([9F 1A]) Terminal Country Code, 2), ((95) Terminal Verification Results, 5),  │
│         │                                                 │ (([5F 2A]) Transaction Country Code, 2), ((9A) Transaction Date, 3), ((9C)      │
│         │                                                 │ Transaction Type, 1), (([9F 37]) Unpredictable Number, 4)])                     │
│ [8D]    │ Card Risk Management Data Object List 2 (CDOL2) │ DOL([((8A) Authorisation Response Code, 2), (([9F 02]) Amount, Authorised, 6),  │
│         │                                                 │ (([9F 03]) Amount, Other (Numeric), 6), (([9F 1A]) Terminal Country Code, 2),   │
│         │                                                 │ ((95) Terminal Verification Results, 5), (([5F 2A]) Transaction Country Code,   │
│         │                                                 │ 2), ((9A) Transaction Date, 3), ((9C) Transaction Type, 1), (([9F 37])          │
│         │                                                 │ Unpredictable Number, 4)])                                                      │
└─────────┴─────────────────────────────────────────────────┴─────────────────────────────────────────────────────────────────────────────────┘
```

**Comment by CharlesButcher** [comment]
Similar result for me with a new Nationwide Visa card. The old one worked fine, but with the new one `emvtool` reports `Error in CAP generation: Application data doesn't include CDOL1 field:`. The listing for `File: 3,2` does have CDOL1, however.

**Comment by russss** [comment]
I think I've fixed this particular problem in issue #20.
